### PR TITLE
Streaming url

### PR DESCRIPTION
**Pull request by rheafernandes**

### CODE REVIEW

File: platform-jobs/samza/image-tagging/src/main/config/local.image-tagging.properties [hunk 1]
```diff
@@ -3,7 +3,7 @@ job.factory.class=org.apache.samza.job.yarn.YarnJobFactory
 job.name=dev.image-tagging
 
 # YARN
-yarn.package.path=file://${basedir}/target/${project.artifactId}-${pom.version}-distribution.tar.gz
+yarn.package.path=file:///Users/rheashalomannfernandes/Sunbird/sunbird-learning-platform/platform-jobs/samza/image-tagging/target/image-tagging-0.0.70-distribution.tar.gz
```
Comment (maheshkumargangula) [inline platform-jobs/samza/image-tagging/src/main/config/local.image-tagging.properties]: Please revert this change.

File: platform-jobs/samza/asset-enrichment/src/main/java/org/ekstep/jobs/samza/task/AssetEnrichmentTask.java [hunk 2]
```diff
@@ -2,22 +2,24 @@
 
 import java.util.Map;
 
+import org.apache.samza.config.Config;
```
[the remark this answers is not in the document]
Comment (amitpriyadarshi) [reply platform-jobs/samza/asset-enrichment/src/main/java/org/ekstep/jobs/samza/task/AssetEnrichmentTask.java]: Remove this import.

File: platform-jobs/samza/asset-enrichment/src/main/java/org/ekstep/jobs/samza/task/AssetEnrichmentTask.java [hunk 3]
```diff
@@ -2,22 +2,24 @@
...
 import org.apache.samza.task.MessageCollector;
+import org.apache.samza.task.TaskContext;
```
[the remark this answers is not in the document]
Comment (amitpriyadarshi) [reply platform-jobs/samza/asset-enrichment/src/main/java/org/ekstep/jobs/samza/task/AssetEnrichmentTask.java]: Remove  this import.
